README.md

```diff
@@ -327,6 +327,46 @@ index
 controller.abort()
 ```
 
+
+### Using Meilisearch behind a proxy <!-- omit in toc -->
+
+#### Custom request config <!-- omit in toc -->
+
+You can customize the request generated ex: add headers
+
+```ts
+const client: MeiliSearch = new MeiliSearch({
+    host: 'http://localhost:3000/api/meilisearch/proxy',
+    requestConfig: {
+        headers: {
+            Authorization: AUTH_TOKEN
```

Comment:
```suggestion
            Authorization: `Bearer ${AUTH_TOKEN}`
```

WDYT ?

Reply:
doesn't have to be a bearer token really this should be custom per user needs as it's just an example